src/XML/src/FMI3/fmi3_xml_dimension_impl.h

```diff
@@ -30,7 +30,7 @@ struct fmi3_xml_dimension_t {
 };
 
 /* declare fmi3_xml_dimension_t vector functions */
-jm_vector_declare_template(fmi3_xml_dimension_t)
+jm_vector_declare_template_no_typedef(fmi3_xml_dimension_t)
```

Comment:
The typedef exists in the public header file, which means that gcc will give a (very long) warning if we add another typedef here when we declare the template.